src/components/AssetMatching/components/Asset/AssetHeading.tsx

```diff
@@ -3,20 +3,32 @@ import Styles from './Asset.style';
 import Icon from 'components/Icon';
 import { AcceptedIconNames } from 'components/Icon/types';
 import { flex } from 'theme/functions';
+import AssetExternalLink, { ExternalLinkProps } from './AssetExternalLink';
 
-interface Props {
+export interface AssetHeadingProps {
   top?: string;
   main: string;
+  externalLink?: ExternalLinkProps;
   bottom?: string;
   iconName: AcceptedIconNames;
 }
-const AssetHeading: FC<Props> = ({ top = '', main, bottom = '', iconName }) => {
+
+const AssetHeading: FC<AssetHeadingProps> = ({
+  top = '',
+  main,
+  bottom = '',
```

Comment:
i dont get why we have these empty strings here? you dont use it

Reply:
Just passing by default empty strings, we can enhance these props with pre-defined text in the future.